jenkins/e2e-image/Dockerfile-1.4

```diff
@@ -15,7 +15,7 @@
 # This file creates a build environment for building and running kubernetes
 # unit and integration tests
 
-FROM gcr.io/google-containers/kubekins-test:v20160822
+FROM gcr.io/k8s-testimages/kubekins-test:1.4-v20161007-49d47fe
```

Comment:
do we want to have two separate `Dockerfile`s?

or do we just want to parameterize the `FROM` line to source from the correct `kubekins-test` image, and then keep everything else the same?

anyone on @kubernetes/test-infra-admins have thoughts?